### DISCUSSION TITLE
multiple session

### DISCUSSION THREAD

**Discussion by GCKeita** [learndevopsonline/learndevopsonline#1919] [Q&A]
Hello, how do I have multiple cloud shell sessions opened on my screen and working simultaneously? (roboshop project)
can anyone help? thanks

**Comment by chaithanya1812** [comment]
Hello @GCKeita , Quickly Join the lab session.
Link:- https://learndevopsonline.my.canva.site/lab-help

**Reply by GCKeita** [reply]
thank you for assisting
